src/CoreWCF.BuildTools/tests/.cs

```diff
@@ -0,0 +1,4 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should this file have a filename and not just an extension? It feels wrong to have a file without a name.

Reply:
I've had an idea, could we have a single source file with this in and have a project Property which triggers adding the common test file to the Compile item group? The Directory.Build.Targets file could add the source file to the Compile item group based on the property value.

Reply:
renamed to XunitTestAssemblyAttribute.xs and included using conditional constructs from msbuild